cgmanifests/cgmanifest.json

```diff
@@ -46,8 +46,8 @@
          "component": {
             "type": "git",
             "git": {
-               "commitHash": "36b48a5707321adba8a70e14da443566a9391e5a",
-               "repositoryUrl": "https://github.com/apache/tvm.git"
+               "commitHash": "e66814704a80bf66ea8c85b95881d88a2ca7a958",
```

Comment:
Is it a permanent change (use of Deelvin instead of Apache)?

Reply:
Yes, we are waiting for merging of our patch on TVM side when return it back to apache/tvm